### ISSUE TITLE
Surgery kits are not working properly after recent patch

### ISSUE THREAD

**Issue by YorgiVonduken** [paulov-t/SIT.Core#135]
**Describe the bug**
Upon receiving a blacked out limb using a surgery kit to completion does not repair the limb and stops all input for movement controls to my character. A circle spins on the surgery kit as though it's still in use. Both CMS and Surv12 seem to cause this issue. It does show the CMS and Surv12 have been decreased by one use on the item but does not restore the limb health.

**To Reproduce**
1. Damage limb to 0 Heath (any method Projectile or Barbed Wire)
2. Use CMS or Surv12 Kit to repair limb
3. Attempt to Move or use inventory items. ( CMS / Surv12  displays spinning circle )

**Expected behavior**
Restore limb health to 1 so further healing can be completed.

**SIT.Core Version:**
1.4.8559.34645

**SPT-Aki Version:**
3.5.7 (Bleeding Edge)

**Additional context**

![Spinning Circle](https://github.com/paulov-t/SIT.Core/assets/110324343/9d88b441-4578-4fda-8dd4-e2889b62237e)

**Comment by zkq631503319** [comment]
Use Surv12 for treat fracture also cause this problem.

**Comment by lllusion-cat** [comment]
When we tried to add injections instead of surgical kits, we found that any attempt to restore a severed limb would make it impossible for the character to move

**Comment by zkq631503319** [comment]
> When we tried to add injections instead of surgical kits, we found that any attempt to restore a severed limb would make it impossible for the character to move

It is really a huge bugs affect playing. We should always use painkillers to avoid limb injury.

**Comment by QQisQQ** [comment]
> When we tried to add injections instead of surgical kits, we found that any attempt to restore a severed limb would make it impossible for the character to move


Yes, using a surgical kit for any action results in the character being unable to move anymore.

**Comment by xjy646** [comment]
I think the bug is caused by the same reason with #56 ,here is my reason:
First, in former releases(SPT-AKI3.5.7 with [SIT.Core.1.4.8550.40954](https://github.com/paulov-t/SIT.Core/releases/tag/Release-75) and earlier), I found that if my friend's stomach was hit and go black, I can hear his character cough, the he ues surgery to repair his stomach and other medical to cure it(to green), In his side he stop cough, but in my side he still cough strongly in the whole raid.
Second, in former releases(SPT-AKI3.5.7 with [SIT.Core.1.4.8550.40954](https://github.com/paulov-t/SIT.Core/releases/tag/Release-75) and earlier), if I was killed by AI in last raid as Client side, I went out and see my health is below 10% (obviously, game feature), I didnt receive cure form Therapist (an uncomplete feature), I just use medical stuff to cure my self completely and join the raid again, the host is my friend, same with the last raid. BUT when I joined I found that my health is still below 10%, as if I never use medical stuff to cure my self. As if the host has stored my health information and denied the data I send to server side.
Third, in #56 , I do more test in (SPT-AKI3.5.7 with [SIT.Core.1.4.8550.40954](https://github.com/paulov-t/SIT.Core/releases/tag/Release-75) and earlier), and found one important feature: if Client side use weapons with very hihg DPS, Client side can kill idel AI(AI already been killed by host through reduce cheast health to zero) successfuly(with significantly more shoots than actually needed), BUT if Client use singlefiremode to shoot AI, the idel AI can never be killed by Client, as if some data keep telling Client that the chest health of AI is not Altered.
Maybe you can consider some more stable way to sync health in different Client and host? Maybe you need some cache data in HOST to record every damage everyone generated and send it to every Client? @paulov-t

**Comment by xjy646** [comment]
One more thing, I think this kind of sync problem is related to latency compensation.

**Comment by H4ck001** [comment]
这个问题已经有私服解决了

**Comment by Sayoki-Yukina** [comment]
> > 这个问题已经有私服解决了
> 
> Please speak in English!

He mean, this bug in some private server was soluted.

**Comment by Sayoki-Yukina** [comment]
> > > > 这个问题已经有私服解决了
> > > 
> > > 
> > > Please speak in English!
> > 
> > 
> > He mean, this bug in some private server was soluted.
> 
> But his answer doesn't help the problem (= _ =)

=.=
